package-build.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kctrl.carvel.dev/v1alpha1
+kind: PackageBuild
+metadata:
+  creationTimestamp: "2022-07-21T18:41:22Z"
```

Comment:
I think we can remove creationTimestamp.

Reply:
Removed.